spec/system/task_spec.rb

```diff
@@ -1,24 +1,25 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
```

Comment:
設定ファイルに追記することで、省略記法で書くことができます。
```suggestion
  let(:project) { create(:project) }
```

Reply:
5f10a69